### DISCUSSION TITLE
Where are save files (`.sol`) located on macOS?

### DISCUSSION THREAD

**Discussion by rafrafek** [ruffle-rs/ruffle#15177] [Q&A] [answered]
Where are save files (`.sol`) located on macOS when using Ruffle Mac Application?

**Comment by danielhjacobs** [comment, marked as answer]
https://docs.rs/dirs/latest/dirs/fn.data_local_dir.html
Examples:
Linux:   `/home/alice/.local/share/ruffle`

macOS: `/Users/Alice/Library/Application Support/ruffle`
Note: *Depending on the level of trust your device attributes to the download, this can be the physical directory or the virtual/sandboxed version of it.*
example:
```
/Users/<username>/Library/Containers/rs.ruffle.ruffle/Data/Library/Application\ Support/ruffle/SharedObjects/localhost/Users/<username>/<gamepath>/<filename>.sol
```

Windows: `C:\Users\Alice\AppData\Local\Ruffle`

**Reply by rafrafek** [reply]
Thank you! I've found my save files!
